tests/test_ht_import_scheduler.rs

```diff
@@ -0,0 +1,208 @@
+
+    let config = create_config(
+        &test_dir,
+        &suite_dir,
+        RCCConfig {
+            binary_path: rcc_binary,
+            profile_config: RCCProfileConfig::Default,
+        },
+    );
+
+    run_scheduler(
+        &test_dir,
+        &config,
+        var("N_SECONDS_RUN_MAX")?.parse::<u64>()?,
+    )
+    .await?;
+
+    assert_working_directory(&config.working_directory).await?;
+    assert_results_directory(&config.results_directory);
+    Ok(())
+}
+
+fn create_config(test_dir: &Utf8Path, suite_dir: &Utf8Path, rcc_config: RCCConfig) -> Config {
+    Config {
+        working_directory: test_dir.join("working"),
+        results_directory: test_dir.join("results"),
+        managed_directory: test_dir.join("managed_robots"),
+        rcc_config,
+        plan_groups: vec![SequentialPlanGroup {
+            plans: vec![PlanConfig {
+                id: "rcc_headless".into(),
+                source: Source::Manual {
+                    base_dir: suite_dir.into(),
+                },
+                robot_config: RobotConfig {
+                    robot_target: "tasks.robot".into(),
+                    top_level_suite_name: None,
+                    suites: vec![],
+                    tests: vec![],
+                    test_tags_include: vec![],
+                    test_tags_exclude: vec![],
+                    variables: vec![],
+                    variable_files: vec![],
+                    argument_files: vec![],
+                    exit_on_failure: false,
+                },
+                execution_config: ExecutionConfig {
+                    n_attempts_max: 1,
+                    retry_strategy: RetryStrategy::Complete,
+                    timeout: 10,
+                },
+                environment_config: EnvironmentConfig::Rcc(RCCEnvironmentConfig {
+                    robot_yaml_path: "robot.yaml".into(),
+                    build_timeout: 1200,
+                    remote_origin: None,
+                    catalog_zip: Some(test_dir.join("hololib.zip")),
+                }),
+                session_config: SessionConfig::Current,
+                working_directory_cleanup_config: WorkingDirectoryCleanupConfig::MaxExecutions(4),
+                host: Host::Source,
+                metadata: PlanMetadata {
+                    application: "app".into(),
+                    suite_name: "minimal_suite".into(),
+                    variant: "".into(),
+                },
+            }],
+            execution_interval: 30,
+        }],
+    }
+}
+
+async fn run_scheduler(
+    test_dir: &Utf8Path,
+    config: &Config,
+    n_seconds_run_max: u64,
+) -> AnyhowResult<()> {
+    let config_path = test_dir.join("config.json");
+    write(&config_path, to_string(&config)?)?;
+    let run_flag_path = test_dir.join("run_flag");
+    write(&run_flag_path, "")?;
+
+    let mut robotmk_no_env_cmd = Command::new("sudo");
+    robotmk_no_env_cmd
+        .arg("unshare")
```

Comment:
Would it make sense to add a small check if internet access as indeed cut, since this is crucial for the test? I was thinking of this:
```
sudo unshare --net -- ping -c 1 github.com && echo "I still have internet access" || [run scheduler]
```

Reply:
I have tested just adding the `.arg` as you have suggested here. I think it will just add `&&` as an argument to `ping`, which fails with:
```
ping: invalid option -- '-'
```
(very cryptic).
I think the most straightforward way would be to add a plan, which will fail, without internet. However, as we discussed, this is overkill here.

Reply:
Actually, I think it is ok to just pass a shell binary in the test (inlining the shell script is not as problematic as I thought it would be.) See new change